banzai/utils/image_utils.py

```diff
@@ -106,36 +110,44 @@ def save_images(pipeline_context, images, master_calibration=False):
     return output_files
 
 
-def get_bpm(image, pipeline_context):
+def load_bpm(image, pipeline_context):
     bpm_filename = dbs.get_bpm(image.telescope_id, image.ccdsum,
```

Comment:
I think this should be `image.telescope.id`

Reply:
I agree